components/support/webextensions/src/main/java/mozilla/components/support/webextensions/facts/WebExtensionFacts.kt

```diff
@@ -37,15 +38,22 @@ private fun emitWebExtensionFact(
     ).collect()
 }
 
-internal fun emitWebExtensionsInitializedFact(extensions: List<WebExtension>) {
-    val installedAddons = extensions.filter { !it.isBuiltIn() }
-    val enabledAddons = installedAddons.filter { it.isEnabled() }
+internal fun emitWebExtensionEnabledFact(extension: WebExtension) {
     emitWebExtensionFact(
         Action.INTERACTION,
-        WebExtensionFacts.Items.WEB_EXTENSIONS_INITIALIZED,
+        WebExtensionFacts.Items.WEB_EXTENSION_ENABLED,
         metadata = mapOf(
-            "installed" to installedAddons.map { it.id },
-            "enabled" to enabledAddons.map { it.id }
+            "enabled" to extension.id
+        )
+    )
+}
```

Comment:
@eliserichards I am a bit concerned by this change since what I believe this code is doing now means we only emit a fact when web extension is enabled while the intent of the original code was to emit all the installed and enabled addons when the web extensions manager is initialized and knowns its current state.

What I think this means is that we won't get the correct list of enabled and installed addons for the metrics ping since this is behaving like an Events ping now where only when an user enables a web extension does this emit a fact.

Reply:
Yes, agree with @gabrielluong that this is a change in semantics. With this we no longer report which add-ons are enabled, but only track users actively `enabling` it which is different. Since add-ons are enabled by default this will rarely happen e.g. only after users disabled and enabled again.

Reply:
@rocketsroger @eliserichards Can you confirm that's a change you wanted ^^. I am thinking you probably wanted the same `enabled` semantics as before. If so, let's revert / change this tomorrow.

Reply:
@csadilek you are correct, this change will track users who are _actively_ enabling an add on. The purpose of separating these two is that we are not getting the correct metrics (see the original issue) when an add on is installed or enabled, and according to product & DS this is the correct behavior. We don't want these as a single event because they are discretely different.

Reply:
@eliserichards Looking for some more clarifications since I originally worked on the original telemetry, Vesta was looking to see the list of installed and enabled addons. Looking at https://github.com/mozilla-mobile/fenix/pull/17445, the intent seems to be to fix has_enabled_addons and has_installed_addons, but I believe this will now report the wrong has_enabled_addons for metrics ping unless the user explicitly toggles on/off an extension.

`emitWebExtensionEnabledFact` also currently only returns a single extension so there's really no need to return a map as its metadata, but we are doing away with reporting the list of enabled addons, which will affect our addons.enabled_addons metric ping for a string list of enabled addons.